### ISSUE TITLE
Escaped Forward Slash Becomes Escaped Backslash with Forward Slash

### ISSUE THREAD

**Issue by niedzielski** [zaach/jsonlint#90]
[json.org](http://www.json.org/) explicitly defines `\/` as a char token. However, `jsonlint <<< '{"foo": "\/"}'` produces `{ "foo": "\\/" }` which is a different value for `foo`.

**Comment by lbegue** [comment]
I don't have a unix/linux shell at hand, though I guess the input string (_keyboard stream_) is first interpreted by the shell and then passed to jsonlint through stdin. 
Hence the 2 control chars \ and / should be escaped first... 
`jsonlint <<< '{"foo":"\\\/"}'`
the shell should then pass this to the stdin of jsonlint as 
`{"foo":"\/"}`
Note: it is not the same when your json is contained in a **file** that is passed to jsonlint...

At least, this test passes: https://jsonlint.com/?json={"foo": "\/"}

**Comment by dl748** [comment]
This is not a shell problem.  Even the browser one does this -> 

{"date":"3\/10\/2016"}

http://zaa.ch/jsonlint/

produces 

{
  "date": "3\\/10\\/2016"
}

but jsonlint.com produces

{
	"date": "3\/10\/2016"
}

**Comment by c22** [comment]
Yes the problem only occurs when you ask jsonlint to reformat. It correctly parses `{"date":"3\/10\/2016"}` as valid JSON, but incorrectly reformats it.

This also affects other escaped characters during reformatting.

`{"path": "C:\\Windows\\system32"}`

Correctly reformats to
```
{
  "path": "C:\\Windows\\system32"
}
```

but `{"path": "C:\\backups"}`

Incorrectly reformats to
```
{
  "path": "C:\backups"
}
```
because `\\b` is being treated as `\` and `\b` instead of `\\` and `b`.

This also affects `\\t` and probably other handling of escaped sequences.
